feeds/wifi-ax/hostapd/files/hostapd.sh

```diff
@@ -97,6 +97,7 @@ hostapd_common_add_device_config() {
 	config_add_boolean country_ie doth
 	config_add_boolean spectrum_mgmt_required
 	config_add_int local_pwr_constraint
+	config_add_int maxassoc
```

Comment:
Using the same variable for both per-SSID and per-Radio.  Was this your intent?
Will give consistent result but reduce flexibility.